package.json

```diff
@@ -402,7 +402,8 @@
     "@rushstack/rig-package": "0.3.11",
     "@rushstack/ts-command-line": "4.11.0",
     "@storybook/react/webpack": "5.72.1",
-    "node-fetch": "2.6.7"
+    "node-fetch": "2.6.7",
+    "protobufjs": "6.11.3"
```

Comment:
Rather than set a resolution to resolve `protobufjs` could we try cleaning its parent dependency from the yarn.lock and adding them again?

``` 
yarn why protobufjs         
└─ centrifuge@npm:2.8.5
   └─ protobufjs@npm:6.11.2 (via npm:^6.11.2)
```

If we check [centrifuge package.json](https://github.com/centrifugal/centrifuge-js/blob/master/package.json#L34) we can see it uses a semver range will pull newer versions of `protobufjs` when installed.

With this in mind we should be able to do:

1. remove `"centrifuge": "2.8.5",` from the package.json
2. run `yarn` to clean the `.lock` file
3. add `"centrifuge": "2.8.5",` back to the package.json
4. run `yarn` again to pull the newer `protobufjs`

Reply:
Fixed!